network.js

```diff
@@ -65,7 +65,26 @@ module.exports = {
     return javaProxyArgs;
   },
   callFetch(dest, opts) {
-    return fetch(dest, opts);
+    return new Promise((resolve, reject)=>{
+      let proxyConfig = proxy.configuration();
+
+      if(Object.keys(proxyConfig).length !== 0 ) {
```

Comment:
@rodrigopavezi I think this should be 

```js
if(Object.keys(proxyConfig).length && proxyConfig.host) {
```

because if proxy.setEndpoint()` is called with a blank string it will reset the proxy config to url.parse("") which results in an object with multiple keys but they'll all be null or `""`.  And proxy.setEndpoint() is called in the player .